integration/swarm/swarm.bats

```diff
@@ -18,25 +20,18 @@ number_of_replicas=1
 timeout=10
 # Retry number for the curl
 number_of_retries=5
-# SHIM PATH
-SHIM_PATH="${SHIM_PATH:-/usr/libexec/kata-containers/kata-shim}"
-# PROXY PATH
-PROXY_PATH="${PROXY_PATH:-/usr/libexec/kata-containers/kata-proxy}"
 
-# This function checks if active processes were
-# left behind by kata-runtime.
+setup() {
+	# Check that processes are not running
+	run check_processes ${PROXY_PATH}
```

Comment:
What I'd really like is for these sanity checks to be run (be)for(e) every test :-) Have a peek at #215 for more info.
Having a quick look at [bats](https://github.com/sstephenson/bats#load-share-common-code), it seems that the `load` command above is just a `source` underneath... in which case.... I think we could write a `bats_common.sh`, that could source `common.sh`, and then run a set of sanity checks - and make all the bats files `load` that common file. That should then have every bats test run the system sanity check.
Actually, I'd like to add that check to both `setup` and `teardown`, but that is harder to do in a nice way that folks won't forget to do for every new bats file.

I'm open to ideas here - but I think with a little more effort we could have gotten these checks, and probably some others (inspired from the mega death soak test) into all the bats files :-)